Source/Core/Core/Movie.cpp

```diff
@@ -72,6 +72,8 @@
 
 // The chunk to allocate movie data in multiples of.
 #define DTM_BASE_LENGTH (1024)
+#define DTM_GC_FRAME_LENGTH (sizeof(ControllerState) + sizeof(u64))
```

Comment:
This and DTM_BASE_LENGTH can be converted into `constexpr size_t` constants (and can also be moved into the Movie namespace)

Reply:
Changes applied. Also noting that I am removing `DTM_BASE_LENGTH` (I didn't create it), as it is never referenced.